scripts/apps/vocabularies/controllers/VocabularyEditController.tsx

```diff
@@ -111,6 +126,60 @@ export function VocabularyEditController(
         return true;
     }
 
+    if ($scope.vocabulary.field_type === EDITOR_BLOCK_FIELD_TYPE) {
+        const vocabulary = $scope.vocabulary;
+
+        $scope.item = {
+            fields_meta: {
+
+                /**
+                 * Fake field, needed for compatibility with sdEditor3 directive
+                 */
+                [editorBlockFieldId]: {draftjsState: vocabulary.field_options?.template},
+            },
+        };
+
+        $scope.editorBlockFormattingOptions = (() => {
+            const excludedOptions: Set<RICH_FORMATTING_OPTION> = new Set<RICH_FORMATTING_OPTION>([
+                'multi-line quote',
+                'comments',
+                'annotation',
+                'suggestions',
+                'table',
+                'media',
+            ]);
+
+            const formattingOptions = Object.entries(getEditor3RichTextFormattingOptions())
+                .map(([notTranslatedOption, translatedOption]) => ({value: [notTranslatedOption, translatedOption]}))
+                .filter(({value}: {value: [RICH_FORMATTING_OPTION, string]}) =>
```

Comment:
When possible you should first filter and only then map, not to waste CPU cycles mapping what you're gonna drop anyway. There wouldn't be performance impact here, but there might be reputational impact if someone checks git blame :D Also it's a good habit to do it the right way so when you write back-end code eventually where it will matter, you wouldn't do such mistakes.